daily-challenges/2024-12/11/lcaloi.29.spec.ts

```
@@ -0,0 +1,43 @@
+    '20k': 20000,
+    '10k': 10000,
+    '5k': 5000,
+    '2k': 2000,
+    '1k': 1000
+};
+
+function doiTien(soTien: number): {} | null {
+    if (soTien < 1000) {
+        return null;
+    }
+    let quantity: number = 0;
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)